core/zms/src/main/rdl/Role.tdl

```diff
@@ -48,6 +48,7 @@ type RoleMeta Struct {
     String userAuthorityFilter (optional); //membership filtered based on user authority configured attributes
     String userAuthorityExpiration (optional); //expiration enforced by a user authority configured attribute
     Int32 groupExpiryDays (optional); //all groups in the domain roles will have specified max expiry days
+    Map<SimpleName,SimpleName> tags (optional); //key-value pair tags
```

Comment:
If this is a map, how are we going to support multiple tags with the same key? I thought we made the index key as role_id+key+value so we can have multiple tags with the same key name -e.g. env=prod,env=stage, etc.